src/lib/comms/include/sol-http.h

```diff
@@ -100,6 +101,9 @@ struct sol_http_param_value {
         struct {
             int value;
         } integer;
+        struct {
+            struct sol_buffer *value;
```

Comment:
const? Or maybe `struct sol_str_slice`? (we can easily get a slice from a buffer)